services/controller/src/plz/controller/results/results_base.py

```diff
@@ -22,6 +28,10 @@ def get(self, execution_id: str) -> ContextManager[Optional['Results']]:
     def is_finished(self, execution_id: str):
         pass
 
+    def compile_metadata(self, execution_id: str, finish_timestamp: int):
```

Comment:
This should probably be marked as private/"protected", or even made a function, not a method.

Reply:
Done